cpp/include/raft/comms/detail/std_comms.hpp

```diff
@@ -140,48 +142,37 @@ class std_comms : public comms_iface {
 
     RAFT_CUDA_TRY(cudaStreamSynchronize(stream_));
```

Comment:
Now that we are using nccl instead of ucp, we should probably avoid any potential deadlocks from dying nccl ranks by using `sync_stream` here instead of the standard cuda synchronization.

Reply:
Updated in latest push